app/__init__.py

```diff
@@ -75,6 +78,58 @@ def create_app():
         app.logger.setLevel(logging.INFO)
         app.logger.info('Facebook Hidden Engagement')
 
+    import click
+    @app.cli.command()
+    @click.option('--filename', default=None)
+    def import_csv(filename):
+        """Import CSV file with DOI's.
+
+        Args:
+            filename: filename with path.
+        """
+        import pandas as pd
+        from app.models import Doi
+
+        if not filename:
+            filename = 'app/static/data/PKP_20171220_100.csv'
```

Comment:
this file seems to be missing

Reply:
yes. can i add it to the repo? its from you.